cisco_secure_web_appliance/README.md

```diff
@@ -0,0 +1,274 @@
+8. Provide Following Details.
+
+    Hostname: \<Datadog-Agent Host Server>
+
+    Port: \<Default Provided>
+
+    Protocol: UDP
+    
+    Maximum message size: \<Valid values for UDP are 1024 to 9216>
+    
+    Facility: \<Default Selected>
+9. Click on Submit. 
+10. Click on Commit Changes at top right-side of Log Subscriptions Page.
+    NOTE: These changes will not go into effect until you commit them.
+
+#### Steps to configure SCP on Remote Server for L4TM Logs.
+
+**Prerequisites:**
+1. Requires SCP Host and Username for the same.
+
+**Configuration:**
+1. Navigate to  System Administration -> Log Subscriptions in Cisco Secure Web Appliance UI.
+2. To add a log subscription for Traffic Monitor Logs, click on  Add `Log Subscription`.  
+3. Select Traffic Monitor Logs as Log Type.
+4. Provide appropriate Log Name.
+5. For FileName, provide a new name or keep the default added name.
+6. Choose SCP on Remote Server as Retrieval Method.
+7. Provide Following Information.
+
+    SCP Host: \<SCP Host IP Address>
+
+    Directory: \<Path to Directory Where Logs would Get Stored>
+    
+    **NOTE:** Make sure that Directory does not have any other log files.
+    SCP Port: \<Default Port>
+    Username: \<SCP Host Username>
+8. Click on Submit. After submitting, SSH key(s)  will get generated. Copy and save both the SSH key(s) as it is only visible once.
+9. Place the given SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
+10. Click on Commit Changes at top right-side of Log Subscriptions Page.
+
+    **NOTE:** These changes will not go into effect until you commit them.
+
+#### Steps to configure SCP on Remote Server for Access Logs.
+
+**Prerequisites:**
+1. Requires SCP Host and Username for the same.
+
+**Configuration:**
+1. Go to System Administration -> Log Subscriptions in Cisco Secure Web Appliance UI.
+2. To add a new log subscription for Access Logs, click Add `Log Subscription`  OR edit already configured Access Logs Subscription.
+3. If you are adding a new subscription then follow the steps 4 to 6 mentioned in the configuration of Syslog Push for Access Logs.
+4. If you are editing existing Access Logs Subscription, select SCP on Remote Server as Retrieval Method.
+5. Provide following information:
+
+    SCP Host: \<SCP Hostname>
+
+    SCP Port: \<Default Provided>
+    
+    Directory: \<Path to store the Log Files>
+    
+    **NOTE:** Make sure that Directory does not have any other log files.
+10. Username: \<SCP Server Username>
+11. Click on Submit. Once you click on `Submit`,  SSH key(s) will get generated. Copy the SSH Key and save it somewhere as this will be only displayed once. 
+12. Place the given SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
+Click on Commit Changes at top right-side of Log Subscriptions Page.
+
+    **NOTE:** These changes will not go into effect until you commit them.
```

Comment:
```suggestion
1. In the Cisco Secure Web Appliance UI, go to System Administration, and then Log Subscriptions.
2. To add a new log subscription for Access Logs, click **Add Log Subscription** or edit an existing Access Logs Subscription.
3. If you are adding a new subscription, then follow steps 4 to 6 mentioned in the Syslog Push for Access Logs section.
4. If you are editing an existing Access Logs Subscription, select SCP on the Remote Server as the Retrieval Method.
5. Provide following information:

    SCP Host: \<SCP Hostname>

    SCP Port: \<Default Provided>
    
    Directory: \<Path to store the Log Files>
    
    **Note:** Make sure that Directory does not have any other log files.
10. Username: \<SCP Server Username>
11. Click **Submit**. Once you click **Submit**,  SSH key(s) are generated. Copy the SSH Key and save it somewhere as this will be only displayed once. 
12. Place the given SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
13. Click **Commit Changes** at top right of Log Subscriptions Page.

    **Note:** These changes do not go into effect until you commit them.
```

Reply:
Updated as per the suggestion.